payments-core/src/main/java/com/stripe/android/model/ConfirmStripeIntentParams.kt

```diff
@@ -27,3 +27,14 @@ sealed interface ConfirmStripeIntentParams : StripeParamsModel, Parcelable {
         internal const val PARAM_MANDATE_DATA = "mandate_data"
     }
 }
+
+internal fun ConfirmStripeIntentParams.createParams(): PaymentMethodCreateParams? {
```

Comment:
This class is public, otherwise I would have considered making bigger changes to support this in the type itself, rather than an extension function.